packages/manager/modules/pci/src/projects/project/project.html

```diff
@@ -84,22 +84,40 @@
                                         </a>
                                     </div>
                                 </div>
-                                <h2
-                                    data-translate="pci_projects_project_documentation"
-                                ></h2>
-                                <ul>
-                                    <li
-                                        data-ng-repeat="link in $ctrl.links track by $index"
+                                <div class="row">
+                                    <div
+                                        class="col-sm-6 col-lg-6 col-xl-6 mb-4"
                                     >
-                                        <a
-                                            data-ng-href="{{ ::link.href }}"
-                                            target="_blank"
-                                            rel="noopener"
-                                            >{{ ::link.translation | translate
-                                            }}</a
+                                        <h2
+                                            data-translate="pci_projects_project_documentation"
+                                        ></h2>
+                                        <ul>
+                                            <li
+                                                data-ng-repeat="link in $ctrl.links track by $index"
+                                            >
+                                                <a
+                                                    data-ng-href="{{ ::link.href }}"
+                                                    target="_blank"
+                                                    rel="noopener"
+                                                    >{{ ::link.translation |
```

Comment:
Not directly brought by this Pull Request but maybe we can consider replacing `{{` syntax and prefer using `translate` directive instead.